cloud-sql/mysql/mysql/package.json

```diff
@@ -12,8 +12,9 @@
     "node": ">=10.0.0"
   },
   "scripts": {
-    "system-test": "mocha test/*.test.js --timeout=60000 --exit",
-    "test": "npm run system-test"
+    "system-test": "mocha test/server.test.js --timeout=60000 --exit",
+    "system-test-unix": "mocha test/server-unix.test.js --timeout=60000 --exit",
```

Comment:
Because these tests juggle environment variables, they need to be run separately to not clobber each other.